test/gtest/plugins/plugins_common.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef __PLUGIN_TEST_H
+#define __PLUGIN_TEST_H
+
+#include <gtest/gtest.h>
+#include <absl/log/check.h>
+#include "backend_engine.h"
+
+namespace plugins_common {
```

Comment:
Namespacing in this PR is inconsistent:

- Global namespace here.
- gtest::obj_plugin in object plugin.
- transferHandler is not namespaced at all.

I don't have strong opinion either way, but please decide on the approach you like and unify all new code accordingly.